spec/services/old_data_removal_service_spec.rb

```diff
@@ -16,4 +16,12 @@
     expect(AnalyticsEvent.count).to eq 1
     expect(CompletedUserJourney.count).to eq 1
   end
+
+  it "deletes old providers, but keeps newer ones" do
```

Comment:
Question -  just to check this test uses the factory, to create 3 rows in the `Provider` table and because one of them is 3 weeks old, it will/should have been deled by `app/services/old_data_removal_service.rb` ?